charts/amazon-cloudwatch-observability/values.yaml

```diff
@@ -403,10 +409,48 @@ manager:
     java:
       repositoryDomain: public.ecr.aws/aws-observability
       repository: adot-autoinstrumentation-java
-      tag: v1.32.2
+      tag: v1.32.3
     python:
       repositoryDomain: public.ecr.aws/aws-observability
       repository: adot-autoinstrumentation-python
+      tag: v0.3.0
+    nodejs:
+      repositoryDomain: 637423224110.dkr.ecr.us-east-1.amazonaws.com/aws-observability
```

Comment:
This should not be exposed right now, please use the public open telemetry nodejs sdk - https://github.com/open-telemetry/opentelemetry-operator/pkgs/container/opentelemetry-operator%2Fautoinstrumentation-nodejs